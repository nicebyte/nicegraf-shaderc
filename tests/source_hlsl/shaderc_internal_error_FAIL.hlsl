//T: polygon ps:PSMain vs:VSMain
//T: polygon_animated ps:PSMain vs:VSMain define:ANIMATE=1

struct PolygonVertex {
  float2 position : SV_POSITION;
  float3 color : ATTR0;
};

PolygonVertex VSMain(PolygonVertex input) {
  PolygonVertex output = input;
  return output;
}

float4 PSMain(PolygonVertex input) : SV_TARGET {
  return float4(input.color, 1.0);
}
