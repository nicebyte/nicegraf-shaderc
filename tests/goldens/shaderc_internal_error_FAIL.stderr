shaderc: internal error: compilation succeeded but failed to optimize: According to the Vulkan spec BuiltIn FragCoord variable needs to be a 4-component 32-bit float vector. ID <31> (OpVariable) has 2 components.
  %input_position = OpVariable %_ptr_Input_v2float Input


